src/test/scala/com.snowplowanalytics.snowplow.scalatracker/TrackerSpec.scala

```diff
@@ -25,36 +25,33 @@ import com.snowplowanalytics.iglu.core.{SchemaKey, SelfDescribingData, SchemaVer
 
 class TrackerSpec extends Specification {
 
-  class TestEmitter extends TEmitter {
+  trait DummyTracker extends Scope {
+    val emitter = new TEmitter {
+      var lastInput = Map[String, String]()
 
-    var lastInput = Map[String, String]()
-
-    def input(event: Map[String, String]) {
-      lastInput = event
+      override def input(event: Map[String, String]): Unit = lastInput = event
     }
+
+    val tracker = new Tracker(List(emitter), "mytracker", "myapp", false)
   }
 
   val unstructEventJson =
     SelfDescribingData[JValue](
-      SchemaKey("com.snowplowanalytics.snowplow", "myevent", "jsonschema", SchemaVer.Full(1,0,0)),
-      ("k1" -> "v1") ~ ("k2" -> "v2"))
+        SchemaKey("com.snowplowanalytics.snowplow", "myevent", "jsonschema", SchemaVer.Full(1, 0, 0)),
+        ("k1" -> "v1") ~ ("k2" -> "v2"))
 
   val contexts = List(
-    SelfDescribingData[JValue](
-      SchemaKey("com.snowplowanalytics.snowplow", "context1", "jsonschema", SchemaVer.Full(1,0,0)),
-      ("number" -> 20)),
-    SelfDescribingData[JValue](
-      SchemaKey("com.snowplowanalytics.snowplow", "context1", "jsonschema", SchemaVer.Full(1,0,0)),
-      ("letters" -> List("a", "b", "c"))))
+      SelfDescribingData[JValue](
+          SchemaKey("com.snowplowanalytics.snowplow", "context1", "jsonschema", SchemaVer.Full(1, 0, 0)),
+          ("number" -> 20)),
+      SelfDescribingData[JValue](
+          SchemaKey("com.snowplowanalytics.snowplow", "context1", "jsonschema", SchemaVer.Full(1, 0, 0)),
+          ("letters" -> List("a", "b", "c")))
```

Comment:
can we indent at two spaces?

Reply:
I removed this extra spaces. I am using http://scalameta.org/scalafmt/ to format the code. I had some problems with code style. If you are interested I can create one more PR configuring scalafmt and also reformatting the project.

Reply:
Great idea! feel free to use a sbt plugin and integrate it into travis if you can 👍

Reply:
Yep, good idea. But @aparra, please include it into a separate commit/PR.